mobile/apps/auth/lib/ui/tools/lock_screen.dart

```diff
@@ -42,7 +45,16 @@ class _LockScreenState extends State<LockScreen> with WidgetsBindingObserver {
     invalidAttemptCount = _lockscreenSetting.getInvalidAttemptCount();
     WidgetsBinding.instance.addObserver(this);
     WidgetsBinding.instance.addPostFrameCallback((timeStamp) {
-      _showLockScreen(source: "postFrameInit");
+      final Object? args = ModalRoute.of(context)?.settings.arguments;
+      if (args is Map && args['manual'] is bool) {
+        _isManualPresentation = args['manual'] as bool;
+      } else {
+        _isManualPresentation = false;
+      }
```

Comment:
You can reduce it to this assuming args is always a Map.
```suggestion
      _isManualPresentation = args?['manual'] as bool? ?? false;
```